docs/source-datastore/scan.md

```diff
@@ -107,20 +107,44 @@ This option enables you to automatically scan file patterns associated with the
 ![record-limit-line](../assets/datastores/scan/record-limit-line-light.png#only-light)
 ![record-limit-line](../assets/datastores/scan/record-limit-line-dark.png#only-dark)
 
-**Step 5:** Click on the **Next** button to Configure the **Enrichment Settings**.
+**Step 5:** Click on the **Next** button to Configure the **Scan Settings**.
 
 ![next-button](../assets/datastores/scan/next-button-light.png#only-light)
 ![next-button](../assets/datastores/scan/next-button-dark.png#only-dark)
 
-**Step 6:** Configure the **Enrichment Settings**.  
+**Step 6:** Configure the **Scan Settings**.
+
+1. **Check Categories:** Users can choose one or more check categories when initiating a scan. This allows for flexible selection based on the desired scope of the operation:
+
+    -  **Metadata**: This option checks that define the expected properties of the table, such as volume. This ensures the table meets required standards during the validation process.
```

Comment:
1. On this part `This option checks` you could replace to `Include checks`.
2. Here you could remove `This ensures the table meets required standards during the validation process.`
3. On final you could add the rule types that this option belongs (It's Volumetric)